packages/@aws-cdk/aws-codebuild/lib/source.ts

```diff
@@ -499,6 +499,14 @@ interface ThirdPartyGitSourceProps extends GitSourceProps {
    * @default every push and every Pull Request (create or update) triggers a build
    */
   readonly webhookFilters?: FilterGroup[];
+
+  /**
+   * This parameter is used for the `url`/`target_url` parameter in the Bitbucket/GitHub commit status.
+   *
+   * @example "The URL that the build will report back to the source provider. Can use built-in CodeBuild variables, like $AWS_REGION"
```

Comment:
This is a very weird example 😃.

```suggestion
   * @example "$CODEBUILD_PUBLIC_BUILD_URL"
```